modules/rank-eval/src/main/java/org/elasticsearch/index/rankeval/DiscountedCumulativeGainAt.java

```diff
@@ -84,6 +86,19 @@ public DiscountedCumulativeGainAt(int position) {
     }
 
     /**
+     * @param position number of top results to check against a given set of relevant results. Must be positive.
+     *  // TODO is there a way to enforce this?
+     * @param normalize If set to true, dcg will be normalized (ndcg)
+     * See https://en.wikipedia.org/wiki/Discounted_cumulative_gain
+     * @param unknownDocRating the rating for docs the user hasn't supplied an explicit rating for
+     * */
+    public DiscountedCumulativeGainAt(int position, boolean normalize, Integer unknownDocRating) {
```

Comment:
Should we move these ctors up in the class so they are all together with the existing one that takes StreamInput as argument?